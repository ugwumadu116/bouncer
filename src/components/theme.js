import * as colors from "./color/color";
import * as fontsize from "./text/fontsize";
import * as fontweight from "./text/fontweight";
import * as lineheight from "./text/lineheight";
import * as letterspacing from "./text/letterspacing";
import * as btnSizes from "./button/button";
import * as btnState from "./button/btnState";
import * as inputsizes from './form/input/inputsizes'
import * as inputbuttonSize from './form/inputButton/inputbuttonsizes'
import * as selectSize from './form/select/selectSizes'


const theme = {
  colors,
  fontsize,
  fontweight,
  lineheight,
  letterspacing,
  btnSizes,
  btnState,
  inputsizes,
  inputbuttonSize,
  selectSize
};
export default theme;
