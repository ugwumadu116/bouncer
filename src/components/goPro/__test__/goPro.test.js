import React from "react";
import { render } from "@testing-library/react";
import theme from "../../theme";
import { ThemeProvider } from "styled-components";
import GoPro from "..";
import '@testing-library/jest-dom/extend-expect'

test("test to match snapshot", async () => {
    const { getByTestId } = render(
      <ThemeProvider theme={theme}>
        <GoPro />
      </ThemeProvider>
    );
    const gopro = getByTestId('gopro')
    expect(gopro).toMatchSnapshot()
});
  
