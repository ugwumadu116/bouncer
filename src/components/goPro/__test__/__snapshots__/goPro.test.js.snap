// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test to match snapshot 1`] = `
<div
  data-testid="gopro"
>
  <div
    class="sc-bxivhb jdDYcm"
  >
    <div
      class="title"
    >
      <p
        class="sc-bdVaJa RsCyL"
      >
        GoPro Hero 6
      </p>
    </div>
    <div
      class="description"
    >
      <p
        class="sc-bdVaJa bchmcq"
      >
        Lorem ipsum dolor amet, consecteture adipisong elit, labor
      </p>
    </div>
    <div
      class="price"
    >
      <p
        class="sc-bdVaJa iNAVdO"
      >
        $299
      </p>
    </div>
    <div
      class="image"
    >
      <img
        alt="GoPro"
        src="https://res.cloudinary.com/decagonbouncer/image/upload/v1575976798/bouncer-frontend/gopro_k0hnnm.png"
      />
    </div>
  </div>
</div>
`;
